readme.md

```diff
@@ -1,5 +1,13 @@
 # Microsoft Azure IoT SDK for .NET
 
+## 2.0.0 clients release notice
+
+A [2.0.0 preview release](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/2022-10-14-v2preview) has been published for each package in this library. 
```

Comment:
If you do want to specify this exact release, then it should be 2.0.0-preview001. And then I'd say keep an eye out for updates.